fapi-client/src/test/scala/demo.sc

```diff
@@ -1,24 +1,38 @@
-import com.amazonaws.auth.BasicAWSCredentials
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.auth.{AWSCredentialsProviderChain, EnvironmentVariableCredentialsProvider, SystemPropertiesCredentialsProvider}
 import com.amazonaws.services.s3.AmazonS3Client
 import com.gu.contentapi.client.GuardianContentClient
 import com.gu.facia.api.FAPI
 import com.gu.facia.client.{AmazonSdkS3Client, ApiClient}
-import concurrent.duration._
 
 import scala.concurrent.Await
+import scala.concurrent.duration._
 
 // demo config
-implicit val capiClient = new GuardianContentClient(apiKey = "API_KEY")
-private val amazonS3Client = new AmazonS3Client(new BasicAWSCredentials("ACCESS_KEY", "SECRET_KEY"))
-implicit val apiClient: ApiClient = ApiClient("aws-frontend-store", "DEV", AmazonSdkS3Client(amazonS3Client))
+val apiKey = "CONTENT_API_KEY"
+val awsProfileName = Some("AWS_PROFILE_NAME")
+
+implicit val capiClient =  new GuardianContentClient(apiKey)
+implicit val apiClient: ApiClient = {
+  val amazonS3Client = {
+    val credentialsProvider = new AWSCredentialsProviderChain(
+      new EnvironmentVariableCredentialsProvider(),
+      new SystemPropertiesCredentialsProvider(),
+      awsProfileName map {
```

Comment:
That means *fill value in here*
Maybe I should call it `FILL_IN_VALUE_HERE`
I see what you mean about the map now! I'll get rid of that.